gatsby-node.ts

```diff
@@ -0,0 +1,16 @@
+// @ts-check
+
+/**
+ * @type {import('gatsby').GatsbyNode['createPages']}
+ */
+
+// eslint-disable-next-line import/no-import-module-exports
+import path from 'path';
+
+exports.createPages = async ({ actions }) => {
+  actions.createSlice({
+    id: 'global-footer',
+    component: path.resolve('./src/components/GlobalFooter/GlobalFooter.tsx'),
+    context: {},
+  });
+};
```

Comment:
This is where I create the Slice for global footer

Reply:
Another ticky bit:
https://www.gatsbyjs.com/docs/how-to/custom-configuration/typescript/#requireresolve

Must use `path.resolve` instead of `require.resolve` for TypeScript